### ISSUE TITLE
Make it possible to generate favicon without padding and without background

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno.resizetizer#190]
## What would you like to be added:

Currently the background of the favicon is a solid color:

![image](https://github.com/unoplatform/uno.resizetizer/assets/1075116/29f64b63-a02a-4b3a-9863-3e2721e1dbf6)

## Why is this needed:

Make the icons nicer and more aligned with other websites

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

**Comment by nickrandolph** [comment]
@MartinZikmund is this on Windows?

**Comment by MartinZikmund** [comment]
Yes seen it in Chefs app in Edge on Windows, but it seems that the solid background is just part of the favicon itself

**Comment by morning4coffe-dev** [comment]
The same should be for the WinUI project also, I think. When I was generating the icon, I needed to manually go to manifest and update it myself with the asset without padding and background.

**Comment by dansiegel** [comment]
@MartinZikmund the scaling is defined in the definition of the base.props. 

https://github.com/unoplatform/uno.templates/blob/c88e0314f56cca5051abea7d7c93d713441ec5c3/src/Uno.Templates/content/unoapp/MyExtensionsApp.1.Shared/base.props#L21

I would assume if you updated this to something like the following you might achieve the result you're looking for:
```xml
<PropertyGroup>
  <UnoIconScale>0.65</UnoIconScale>
  <UnoIconScale Condition="$(MSBuildProjectName.EndsWith('.Wasm')) or $(TargetFramework.Contains('-windows10'))">1.0</UnoIconScale>
</PropertyGroup>
<ItemGroup>
  <UnoIcon Include="$(MSBuildThisFileDirectory)Icons\icon.svg"
        ForegroundFile="$(MSBuildThisFileDirectory)Icons\icon_foreground.svg"
        ForegroundScale="$(UnoIconScale)"
        Color="#00000000" />
</ItemGroup>
```

**Comment by MartinZikmund** [comment]
Will try this out

**Comment by MartinZikmund** [comment]
When I set the color as "#00000000" in `UnoIcon`, the `icon.svg` takes over and makes it solid color anyway. I had to make the `icon.svg` just `<svg></svg>` to make it work, but it feels weird...

**Comment by MartinZikmund** [comment]
Also, how does one configure icon and splash screen in single project?

**Comment by domenic** [comment]
Hi, I was directed to Uno Resizetizer as the best way to generate .ico files and pngs from SVGs. But am I correct that this issue means that if my SVG has a transparent background, Uno Resizetizer will not correctly generate the resulting .icos and PNGs? That's certainly what I'm seeing at the moment, with all my generated files having a black background that I cannot figure out how to get rid of...

**Comment by MartinZikmund** [comment]
> Hi, I was directed to Uno Resizetizer as the best way to generate .ico files and pngs from SVGs. But am I correct that this issue means that if my SVG has a transparent background, Uno Resizetizer will not correctly generate the resulting .icos and PNGs? That's certainly what I'm seeing at the moment, with all my generated files having a black background that I cannot figure out how to get rid of...

My current workaround is as follows:

https://github.com/MartinZikmund/stopwatch/blob/778e6ed4c66771b82b7e53532cad0a7c66367303/src/Stopwatch/Stopwatch.csproj#L32-L36

And then providing two different icon files https://github.com/MartinZikmund/stopwatch/tree/main/src/Stopwatch/Assets/Icons here - one with solid background, one with transparent.

This way mobile targets can use solid background, but other targets can use the transparent one.

We certainly will adjust this in Uno to make it easier for users, but for now this approach works.
